src/features/autosuggest/autosuggest.ts

```diff
@@ -12,5 +12,6 @@ export async function autoSuggest(params: AutosuggestOptions): Promise<SuggestRe
   const queryParams = Object.assign(config, defaults, params);
   const url = buildApiUrl(suggestEndpoint || SUGGEST_ENDPOINT_PROD, queryParams);
 
-  return fetch(url).then((data) => data.json());
+  const data = await fetch(url);
+  return data.json();
```

Comment:
`data.json()` returns a promise (of `any` type).  To be consistent with the usage of `await`, maybe we can do something like:

`return await data.json() as SuggestResponse;`

I added the typing at the end because whenever I omit the typing with `.json()` when using `fetch` API in typescript, the linter or typescript check on my IDE always complaints about the return types.

Reply:
interesting, let me check that in an actual project to see what typescript says on the return type (since the function itself is typed as having a return Promise<SuggestResponse> type)